index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+     <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+
+     </head>
+     <body>
+       <header>
+         <div class="head-wrapper">
+           <nav class="nav-bar">
+             <ul>
+               <li class="logo"><a href="#">My Logo</a></li>
+            </ul>
+            <div class="menu">
+                   <div class="toogle-btn"></div>
+                   <div class="toogle-btn"></div>
+                   <div class="toogle-btn"></div>
+             </div>
+           </nav>
+         </div>
+       </header>
+
+       <section>
+         <div class="headline-wrapper">
+          <div class="headline-container">
+            <div class="intro">I'm Vana Glad to see you!</div>
+            <div class="message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+            <div class="end-message">
+              <h3>LET'S CONNECT</h3>
+            </div>
+          </div>
+
+          <div class="social-icon">
+            <div class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-medium"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-github"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></div>
```

Comment:
- [ ]  Links should be inside `<ul> <ul>` tags let's go and put them inside as the first review suggested. Kindly follow this skeleton
````
<ul>
  <li> <a href="#"><img src="img/icon/twitter.svg" alt="Twitter Icon"></a>  </li>
  <li> <a href="#"><img src="img/icon/linkedin.svg" alt="Linkedin Icon"></a>  </li>
  ...
</ul>
```

Reply:
when I put them in this form all my logos are turning into 1 line vertically.